src/commands/tick-log.js

```diff
@@ -40,10 +43,12 @@ function log(argv) {
   let { start, end } = parseDateRange(argv.date)
   start = moment(start).toArray()
   end = moment(end).toArray()
+  const filter = argv.filter ? compileFilter(argv.filter) : null 
+  const opts = filter ? { filter } : { start, end, tags: argv.tag }
```

Comment:
Actually, after looking at this more closely I see that I need to change `query.findEntries` to allow `start`, `end`, and `filter` (which was originally a restriction when I allowed dates in the filter). Thanks for drawing my eyes to this